### PR TITLE
[NEW] Custom Status

### LINKED ISSUE #1536: Set a custom status message
- Your Rocket.Chat Experimental app version: 1.7.0
- Your Rocket.Chat server version: 2.3.1
- Device (or Simulator) you're running with: iOS/Android

Users cannot set a custom status message on mobile like they can in the browser.

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
@RocketChat/ReactNative

Closes #1536 .

- [X] Show customStatus on RoomActions
- [X] Show customStatus on RoomInfo
- [X] Show customStatus as a Banner
- [x] Edit customStatus on Profile
- [x] Edit customStatus from SideBar
- [x] Handle stream changes

Screen Shot 2020-03-03 at 11 16 55

Screen Shot 2020-03-03 at 11 17 00

Screen Shot 2020-03-03 at 11 17 15

Screen Shot 2020-03-04 at 12 52 19

Based on #1176.

### CODE REVIEW

File: app/containers/MessageBox/UploadModal.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ class UploadModal extends Component {
 				hideModalContentWhileAnimating
 				avoidKeyboard
 			>
-				<View style={[styles.container, { width: width - 32, backgroundColor: themes[theme].chatComponentBackground }, split && sharedStyles.modal]}>
+				<View style={[styles.container, { width: width - 32, backgroundColor: themes[theme].chatComponentBackground }, split && [sharedStyles.modal, sharedStyles.modalFormSheet]]}>
```
Comment (djorkaeffalexandre) [inline app/containers/MessageBox/UploadModal.js]: A minimal fix on size of upload modal.

File: app/views/SidebarView/CustomStatus.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
+import React, { Component } from 'react';
```
Comment (djorkaeffalexandre) [inline app/views/SidebarView/CustomStatus.js]: Instead use modal, try to use `react-native-android-prompt`.

**Comment by wreiske** [comment]
Woo! 🎉

**Comment by wreiske** [comment]
👀 excited 🚀

**Comment by wreiske** [comment]
🎈

**Pull request merged**